index.js

```diff
@@ -249,7 +255,16 @@ class ServerlessCustomDomain {
       throw new Error(`${action} is not a valid action. action must be either CREATE or DELETE`);
     }
 
+    if (this.serverless.service.custom.customDomain.createRoute53Record !== undefined
```

Comment:
Just defensive programming.  The `createRoute53Record` property won't exist in `this.serverless.service.custom.customDomain` if not defined in the `serverless.yml` and therefore won't strict equal false.